dev/tidb-in-kubernetes/maintain/tidb-binlog.md

```diff
@@ -32,7 +32,74 @@ category: how-to
 
         * Apache Kafka：通过将 `binlog.drainer.destDBType` 设置为 `kafka` 来启用。同时，必须在 `binlog.drainer.kafka` 中配置目标集群的 zookeeper 地址和 Kafka 地址。
 
-2. 创建一个新的 TiDB 集群或更新现有的集群：
+2. 为 TiDB 与 Pump 组件设置亲和性和反亲和性：
+
+    > **注意：**
+    >
+    > 如果在生产环境中开启 TiDB Binlog，建议为 TiDB 与 Pump 组件设置亲和性和反亲和性。如果在内网测试环境中尝试使用开启 TiDB Binlog，可以跳过此步。
+
+    默认情况下，TiDB 的 affinity 亲和性设置为 `{}`。由于目前 Pump 组件与 TiDB 组件默认并非一一对应，当启用 TiDB Binlog 时，如果 Pump 与 TiDB 组件分开部署并出现网络隔离，而且 TiDB 组件还开启了 `ignore-error`，则会导致 TiDB 丢失 Binlog。推荐通过亲和性特性将 TiDB 组件与 Pump 部署在同一台 Node 上，同时通过反亲和性特性将 Pump 分散在不同的 Node 上，每台 Node 上至多仅需一个 Pump 实例。
+
+    * 将 `tidb.affinity` 按照如下设置：
+
+      ```yaml
+      tidb:
+        affinity:
+          podAffinity:
+            requiredDuringSchedulingIgnoredDuringExecution:
+              - labelSelector:
+                  matchExpressions:
+                    - key: "app.kubernetes.io/component"
+                      operator: In
+                      values:
+                        - "pump"
+                    - key: "app.kubernetes.io/managed-by"
+                      operator: In
+                      values:
+                        - "tidb-operator"
+                    - key: "app.kubernetes.io/name"
+                      operator: In
+                      values:
+                        - "tidb-cluster"
+                    - key: "app.kubernetes.io/instance"
+                      operator: In
+                      values:
+                        - <release-name>
+                topologyKey: kubernetes.io/hostname
+      ```
+
+    * 将 `binlog.pump.affinity` 按照如下设置：
+
+      ```yaml
```

Comment:
Indent this code block with 8 spaces from the beginning of each line.

Reply:
updated